test/uk/gov/hmrc/cardpaymentfrontend/controllers/PaymentFailedControllerSpec.scala

```diff
@@ -62,6 +62,20 @@ class PaymentFailedControllerSpec extends ItSpec {
         langToggleText should contain theSameElementsAs List("English", "Newid yr iaith ir Gymraeg Cymraeg")
       }
 
+      "show the Title tab correctly in English" in {
+        PayApiStub.stubForFindBySessionId2xx(TestJourneys.PfSa.journeyAfterFailWebPayment)
+        val result = systemUnderTest.renderPage(fakeGetRequest)
+        val document = Jsoup.parse(contentAsString(result))
+        document.title shouldBe "Payment failed - Pay your Self Assessment - GOV.UK"
```

Comment:
we should get a test in on an action that has form validation to check for the scenario where the title gets prepended with `Error: xxx - Pay your self assessment - gov.uk`